framework/branch-key-store.md

```diff
@@ -575,54 +741,71 @@ and the `branchKeyId` from the returned `branch-key-id` field.
 
 This operation MUST return the constructed [beacon key materials](./structures.md#beacon-key-materials).
 
-## Encryption Context
-
-This section describes how the AWS KMS encryption context is built
-from the DynamoDB items that store the branch keys.
+## Branch Key Context
 
-The following encryption context keys are shared:
+Branch Key Context is a set of key value pairs (a Map) that contain contextual information about the Branch Key Item.
 
-- MUST have a `branch-key-id` attribute
-- The `branch-key-id` field MUST not be an empty string
-- MUST have a `type` attribute
-- The `type` field MUST not be an empty string
-- MUST have a `create-time` attribute
-- MUST have a `tablename` attribute to store the logicalKeyStoreName
-- MUST have a `kms-arn` attribute
-- MUST have a `hierarchy-version`
-- MUST NOT have a `enc` attribute
+The Branch Key Context:
 
-Any additionally attributes on the DynamoDB item
-MUST be added to the encryption context.
+- MUST have a `branch-key-id` key who's value MUST not be an empty string
+- MUST have a `type` key who's value MUST not be an empty string
+- MUST have a `create-time` key in ISO 8601 format in UTC
+- MUST have a `tablename` key who's value is the logicalKeyStoreName
+- MUST have a `kms-arn` key who's value is valid KMS ARN
+- MUST have a `hierarchy-version` key who's value is either "1" or "2"
+- MUST NOT have a `enc` key
+- MAY have one or more keys prefixed with `aws-crypto-ec:` which is the encyption context send by the customer.
+  The `aws-crypto-ec:` prefix is prepended by the library
 
-### ACTIVE Encryption Context
+### ACTIVE Branch Key Context
 
 The ACTIVE branch key is a copy of the DECRYPT_ONLY with the same `version`.
 It is structured slightly differently so that the active version can be accessed quickly.
 
-In addition to the [encryption context](#encryption-context):
+In addition to the [branch key context](#branch-key-context):
 
-The ACTIVE encryption context value of the `type` attribute MUST equal to `"branch:ACTIVE"`.
-The ACTIVE encryption context MUST have a `version` attribute.
-The `version` attribute MUST store the branch key version formatted like `"branch:version:"` + `version`.
+The ACTIVE branch key context value of the `type`key MUST equal to `"branch:ACTIVE"`.
+The ACTIVE branch key context MUST have a `version` key.
+The `version` key MUST store the branch key version formatted like `"branch:version:"` + `version`.
```

Comment:
Issue: This is almost certainty my fault, but we should pick one format for `version`. I think `<version UUID>` is more descriptive than `version`.

```suggestion
The `version` key MUST store the branch key version formatted like `"branch:version:"` + `<version UUID>`.
```

Reply:
Did it in some commit I can't remember.

https://github.com/awslabs/aws-encryption-sdk-specification/pull/297/files#diff-c6735a6d874d8b9c52040541b81908877a6b9965b51d5b443c35af9895ec8676R771-R773